docs/xjit.md

```diff
@@ -67,6 +68,16 @@ These parameters can be used to modify the behavior of `-Xjit`:
 
 : From z/OS V2R3, residency mode for 64-bit applications (RMODE64) is enabled by default. This feature allows the JIT to allocate executable code caches above the 2 GB memory bar, which is the default behavior. Use this option to turn off this JIT behavior.
 
+### `enableGPU`
+
+**(Windows (x86-64) or Linux (x86-64 and IBM POWER 8))**
+
+        -Xjit:enableGPU
+
+: Enables the JIT compiler to offload certain processing tasks to a graphics processing unit (GPU). The JIT determines which functions to offload based on performance heuristics. Systems must support NVIDIA Compute Unified Device Architecture (CUDA).  The JIT requires the CUDA Toolkit 7.5 and your GPU device must have a minimum compute capability of 3.0
+
+: To troubleshoot operations between the JIT compiler and the GPU, use `-Xjit:enableGPU={verbose}`, which provides output showing the processing tasks that are offloaded and their status. To send this output to a file (`output.txt`), run `-Xjit:enableGPU={verbose},vlog=output.txt` when you start your application.
```

Comment:
I think previously we exposed more suboptions, e.g. 'enforce', maybe some others...

Please note that we were planning to publish a blog on GPU support in JIT. It definitely mentions a few more sub-options..

Reply:
Yes we did ... `enforce` was in the diagnostics section. I've added it. Not aware of any further options.

Reply:
Ok, let's keep just these for now.

Reply:
OK great. Are you able to approve your review please?